modules/installation-bare-metal-agent-installer-config-yaml.adoc

```diff
@@ -76,6 +76,41 @@ Class E CIDR range is reserved for a future use. To use the Class E CIDR range,
 <9> The cluster network provider Container Network Interface (CNI) plug-in to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN` .
 <10> The IP address pool to use for service IP addresses. You can enter only one IP address pool. This block must not overlap with existing physical networks. If you need to access the services from an external network, configure load balancers and routers to manage the traffic.
 <11> You must set the platform to `none` for a single-node cluster. You can also set the platform to `vSphere` and `baremetal`.
++
+[NOTE]
+====
+When you set the platform to `vSphere` or `baremetal`, you can configure IP address endpoints for cluster nodes in three ways:
+
+* IPv4
+* IPv6
+* IPv4 and IPv6 parallelly (dual-stack)
+
+.Example of dual-stack networking
+[source,yaml]
+----
+networking:
+  clusterNetwork:
+    - cidr: 172.21.0.0/16
+      hostPrefix: 23
+    - cidr: fd02::/48
+      hostPrefix: 64
+  machineNetwork:
+    - cidr: 172.23.0.0/16
+    - cidr: 2001:DB8::/32
+  networkType: OVNKubernetes
```

Comment:
Nit - recommend moving the networkType below the serviceNetwork so that the all of the network definitions are together.

Reply:
implemented